plugins/linux/nsplugin/microservices.go

```diff
@@ -241,6 +241,7 @@ func (plugin *NsHandler) trackMicroservices(ctx context.Context) {
 		select {
 		case <-timer.C:
 			if err := plugin.dockerClient.Ping(); err != nil {
+				plugin.log.Error(err)
```

Comment:
The logging of error this way was omitted to prevent spamming of errors on each interval. The error is logged only when the client was ok before, see in next lines.

Reply:
this was only for debugging and I forgot to remove it, sorry